### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by TimofeiShorokhov**
Refactoring function for changing order status

### CODE REVIEW

File: dao/OrderRepository.go [hunk 1]
```diff
@@ -78,11 +78,28 @@ func (r *OrderPostgres) GetActiveOrderFromDB(id int) (Order, error) {
...
+	insertValue := `Select delivery_service_id,id,courier_id,delivery_time,customer_address,status,order_date,restaurant_address,picked from delivery where id = $1`
+	get, err := r.db.Query(insertValue, id)
+	if err != nil {
+		log.Println("Error with getting order by id: " + err.Error())
+		return Order{}, err
+	}
+
+	for get.Next() {
+		var order Order
+		err = get.Scan(&order.IdDeliveryService, &order.Id, &order.IdCourier, &order.DeliveryTime, &order.CustomerAddress, &order.Status, &order.OrderDate, &order.RestaurantAddress, &order.Picked)
```
Comment (Baraulia) [inline dao/OrderRepository.go:93]: Why do you use r.db.Query? If you expect to get one object, you can use r.db.QueryRow and you don't need to use then get.Next().

**Review by Baraulia — APPROVED** [review]

**Review by Nalivayko13 — COMMENTED** [review]
LGTM

**Comment by TonySinero** [comment]
+

**Comment by TonySinero** [comment]
Reviewed the changes

**Pull request merged**